tests/functional/simple_test.py

```diff
@@ -383,25 +393,26 @@ def test_cant_wheel_package(tmpdir):
 
         out, err = venv_update()
         err = strip_pip_warnings(err)
-        assert err == ''
-
+        assert err == '''\
+  Failed building wheel for cant-wheel-package
+SLOW!! no wheel found after building (couldn't be wheeled?): cant-wheel-package==0.1.0
+  Failed building wheel for cant-wheel-package
+'''
         out = uncolor(out)
 
         # for unknown reasons, py27 has an extra line with four spaces in this output, where py26 does not.
         out = out.replace('\n    \n', '\n')
-        assert '''
-
-----------------------------------------
-Failed building wheel for cant-wheel-package
-Running setup.py bdist_wheel for pure-python-package
-Destination directory: %s/home/.cache/pip-faster/wheelhouse''' % tmpdir + '''
-SLOW!! no wheel found after building (couldn't be wheeled?): cant-wheel-package==0.1.0
-Installing collected packages: cant-wheel-package, pure-python-package
-  Running setup.py install for cant-wheel-package
-  Could not find .egg-info directory in install record for cant-wheel-package (from -r requirements.txt (line 1))
-Successfully installed cant-wheel-package pure-python-package
-Cleaning up...
-''' in out  # noqa
+        # The ... in the logs are generated with a spinner, which also leaves these
+        # \x08 (backspaces) around
+        out = out.replace('\x08', '')
+        assert all([line in out for line in [
```

Comment:
This is a much weaker assertion than before.
Ordering isn't asserted, and these snippets aren't necessarily complete lines.